roles/openshift_node_dnsmasq/tasks/network-manager.yml

```diff
@@ -1,9 +1,32 @@
 ---
+
+- name: Configured dnsmasq interfaces
+  lineinfile:
+    path: /etc/sysconfig/atomic-openshift-node
+    line: "OPENSHIFT_NODE_DNSMASQ_INTERFACES={{ openshift_node_dnsmasq_interfaces | split(',') | join (',') }}"
+    regexp: "^OPENSHIFT_NODE_DNSMASQ_INTERFACES=.*"
+  notify: restart NetworkManager
+  when: openshift_node_dnsmasq_interfaces is defined
+
 - name: Install network manager dispatch script
   copy:
     src: networkmanager/99-origin-dns.sh
     dest: /etc/NetworkManager/dispatcher.d/
     mode: 0755
   notify: restart NetworkManager
 
+- name: Ensure dnsmasq can read /etc/origin/node/resolv.conf
```

Comment:
I would think that dnsmasq is selinux contained.  Probably need to sefcontext these files.